modules/imgproc/test/test_contours_new.cpp

```diff
@@ -446,10 +447,8 @@ TEST_P(Imgproc_FindContours_Modes2, new_accuracy)
         for (int x = 1; x < sz.width - 1; ++x)
         {
             if (img.at<uchar>(y, x) != 0
-                && (img.at<uchar>(y - 1, x) == 0
-                    || img.at<uchar>(y + 1, x) == 0
-                    || img.at<uchar>(y, x + 1) == 0
-                    || img.at<uchar>(y, x - 1) == 0))
+                && (img.at<uchar>(y - 1, x) == 0 || img.at<uchar>(y + 1, x) == 0
+                    || img.at<uchar>(y, x + 1) == 0 || img.at<uchar>(y, x - 1) == 0))
```

Comment:
the same, align `.at()` calls one per line

Reply:
Can not make clang-format to align continuations properly when breaks are before binary operators, perhaps it is some sort of a bug. So I decided to switch to `BreakBeforeBinaryOperators: None` in the end :slightly_frowning_face: This code will be:
```.cpp
            if (img.at<uchar>(y, x) != 0 &&
                ((img.at<uchar>(y - 1, x) == 0) || (img.at<uchar>(y + 1, x) == 0) ||
                 (img.at<uchar>(y, x + 1) == 0) || (img.at<uchar>(y, x - 1) == 0)))
            {
```